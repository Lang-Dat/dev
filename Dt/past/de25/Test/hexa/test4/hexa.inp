101110100001010011000101000001110110010011110010111111100
1001101001000001111011101110010110100
01010010101111010011101110101011110000001100011010010010101110110001010
10001110111110001111101111010101110011101000110111111001011011100101010000
1010101111
00000111111111111001001110110110111011011101001010000010100011000010
1101000010111011000111010001001010101000101010111011010001000111001100101111100010010000001000
10101111010010111011101010
010110001110100111100000111011000110011011011101
00100110000111000001000101101001011010001111011001001100100110011101100101001100
