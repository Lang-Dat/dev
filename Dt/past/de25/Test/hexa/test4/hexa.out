174298A0EC9E5FC
13483DDCB4
295E9DD5E063495D8A
23BE3EF573A37E5B950
2AF
7FF93B6EDD2828C2
342EC744AA2AED11CCBE2408
2BD2EEA
58E9E0EC66DD
261C116968F64C99D94C
